gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -153,7 +153,17 @@
   public static final String EXTRACT_PRIMARY_KEY_FIELDS_KEY = "extract.primary.key.fields";
   public static final String EXTRACT_DELTA_FIELDS_KEY = "extract.delta.fields";
   public static final String EXTRACT_SCHEMA = "extract.schema";
-  public static final String EXTRACT_PULL_LIMIT = "extract.pull.limit";
+  public static final String EXTRACT_LIMIT_ENABLED_KEY = "extract.limit.enabled";
```

Comment:
How about putting the config names in the LimiterFactory class? Then it is very clear where they are used.